code/modules/client/preferences_savefile.dm

```diff
@@ -375,7 +383,7 @@ SAVEFILE UPDATING/VERSIONING - 'Simplified', or rather, more coder-friendly ~Car
 	facial_hair_color			= sanitize_hexcolor(facial_hair_color, 3, 0)
 	eye_color		= sanitize_hexcolor(eye_color, 3, 0)
 	skin_tone		= sanitize_inlist(skin_tone, skin_tones)
-	backbag			= sanitize_integer(backbag, 1, backbaglist.len, initial(backbag))
+	backbag			= sanitize_inlist(backbag, backbaglist, initial(backbag))
```

Comment:
@MrStonedOne 

Yes, it is.
But do you know where ```update_character()``` is called? *IN* ```load_path()```, a few lines above this line :P